src/main/java/org/greenbuttonalliance/gbaresourceserver/usage/controller/IntervalBlockController.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2022 Green Button Alliance, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
```

Comment:
Following the existing convention, I added this license to all the Java files I created. Is this something I should keep doing?

Reply:
Yes.  You can define it using the IntelliJ copyright text editor if you haven't done so already.